CynaraCosta/OrganizArt
# created Product and Sales Models and created database filled with Produtcs and Sales

## What I did
1. Product Model
2. Sale Model
3. Log Product Function
4. Log Sale Function

## How to test
1. Check if the info organization makes sense
2. You can try: print(loggingProducts) inside the viewDidLoad at the ViewController to check if it prints the correct Array of products
3. Check if the data type in each Model is correct
